### ISSUE TITLE
Workflow doesn't recognize volume format change

### ISSUE THREAD

**Issue by yagmot** [grahamgilbert/imagr#210]
### Description of Issue/Question
Example: app opens and workflows are loaded. At this time, the target volume is formatted HFS. Choose a workflow that erases the volume and formats as APFS, then attempts to restore an APFS image. After the volume is erased, the workflow complains that you are trying to restore an APFS image to an HFS volume.

Tried splitting the eraseVolume step into a separate workflow that is included because I read in another thread that volumes are supposedly rescanned when a workflow is completed, but that does not seem to be the case for included workflows.

### Setup
[testconfig2.txt](https://github.com/grahamgilbert/imagr/files/2012218/testconfig2.txt)

### Steps to Reproduce Issue
- Boot into imagr
- format volume as HFS
- reload workflows
- launch workflow

### Versions Report
1.4.1
NBI is 10.13

NBI is 10.13.

**Comment by grahamgilbert** [comment]
Your nbi is 10.13.0? Details matter.

**Comment by grahamgilbert** [comment]
@mosen looks like things you've been working on. Is this fixed in master?

**Comment by mosen** [comment]
Yes I have a fix but it specifically relates to using a script item to reformat the drive. Let me check if it will rescan volumes using erasevolume

**Comment by yagmot** [comment]
My NBI is 10.13.3 17D47
